docs/source/actions.rst

```diff
@@ -140,3 +140,26 @@ You can add fields to both the start message and the success message of an actio
 
 If you want to include some extra information in case of failures beyond the exception you can always log a regular message with that information.
 Since the message will be recorded inside the context of the action its information will be clearly tied to the result of the action by the person (or code!) reading the logs later on.
+
+
+.. _extract errors:
+
+Error Handling
+--------------
+
+When an action fails with an exception a failed action message is logged.
+You can customize which fields are included in this message by registering a callable that converts exceptions into fields.
+
+For example, the following registration means all failed actions that fail with a ``MyException`` will have a ``code`` field in the action end message:
+
+.. code-block:: python
+
+   class MyException(Exception):
+       def __init__(self, code):
+           self.code = code
+
+   from eliot import extract_fields_for_failures
+   extract_fields_for_failures(MyException, lambda e: {"code": e.code})
```

Comment:
:question: `extract_fields_for_failures` sounds like a function that'd return something rather than a way of registering a logging handler. Would you consider calling it `register_exception_handler` or something?